Formula/pari-galdata.rb

```diff
@@ -0,0 +1,21 @@
+class PariGaldata < Formula
+  desc "Galois resolvents data for PARI/GP"
+  homepage "https://pari.math.u-bordeaux.fr/packages.html"
+  url "https://pari.math.u-bordeaux.fr/pub/pari/packages/galdata.tgz"
+  # Refer to http://pari.math.u-bordeaux.fr/packages.html#packages for most recent package date
+  version "20080411"
+  sha256 "b7c1650099b24a20bdade47a85a928351c586287f0d4c73933313873e63290dd"
+  license "GPL-2.0-or-later"
+
```

Comment:
```suggestion

livecheck do
    url :homepage
    regex(%r{>galdata\.t[^<]+?</a>(?:\s|&nbsp;)*?(?:\([\d.]+ \w+\),\s*)?([a-zA-Z]+ \d{1,2} \d{4})}i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| Date.parse(match.first)&.strftime("%Y%m%d") }
    end
  end

```

This is just a quick proof of concept to further the discussion and I'm going through the regex to see if there are any areas that can be refined, so please don't commit it yet. Once we've settled on a final regex, I'll add suggestions for the other formulae (should be the same except for the tarball filename) and we can commit them all then.

Reply:
After some refinement, here's a slightly looser variation of the regex that may be better:

```ruby
regex(%r{>\s*galdata\.t[^<]+?</a>(?:[&(;\s\w]+?(?:\),?|,))?\s*([a-z]+ \d{1,2} \d{4})}i)
```

This matches the current text format (i.e., `>galdata.tgz</a>&nbsp;(52 KB), Apr 11 2008`) but it also matches some reasonable variations while still working as expected.